doc/modules/neighbors.rst

```diff
@@ -834,8 +852,9 @@ Here the ``transform`` operation returns :math:`LX^T`, therefore its time
 complexity equals ``n_components * n_features * n_samples_test``. There is no
 added space complexity in the operation.
 
-
-.. topic:: References:
+|details-start|
+**References**
+|details-split|
```

Comment:
As mentioned in https://github.com/scikit-learn/scikit-learn/pull/27919#discussion_r1487583725, this dropdown contains a reference from it's exterior. We'll have to revert it.

```suggestion

.. topic:: References:
```